src/Model/ModelData.php

```diff
@@ -601,41 +485,6 @@ public function hasValue(string $field, array $arguments = [], bool $cache = tru
         return (bool) $result;
     }
 
-    /**
-     * Get the string value of a field on this object that has been suitable escaped to be inserted directly into a
-     * template.
-     *
-     * @deprecated 5.4.0 Will be removed without equivalent functionality to replace it
-     */
-    public function XML_val(string $field, array $arguments = [], bool $cache = false): string
```

Comment:
This used to be used by the template engine, but it's not anymore and there was only one other place that used it and that wasn't even called itself. So it's not needed anymore.